testdata/p4_16_samples/header-bool-bmv2.p4

```diff
@@ -14,8 +15,22 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-#include <core.p4>
+#include <v1model.p4>
 
-header boolfield {
-    bool x;
+header hdr {
+    bit<31> f;
+    bool    x;
 }
+
+control compute(inout hdr h) {
+    apply {
+        hdr tmp;
```

Comment:
your indentation is inconsistent.

Reply:
wow, sorry, I missed fixing the indentation for the new test.  I will fix the other one too.

Reply:
Is it?  The 'hdr tmp;' is indented by 4 spaces from the 'apply'.   Each data struct member is also indented by 4 spaces.